### PR TITLE
[8.0] Move make_invoice_by_partner out of makeInvoices method to improve possibility of inheritance

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Move make_invoice_by_partner out of makeInvoices method to improve possibility of inheritance. And add context propagation.

### CODE REVIEW

**Comment by adrienpeiffer** [comment]
Any news about this PR ?

**Comment by adrienpeiffer** [comment]
Any news about this PR ?

**Comment by lmignon** [comment]
Any nens about this PR?

**Comment by adrienpeiffer** [comment]
OPW 645036

File: addons/purchase/wizard/purchase_line_invoice.py [hunk 1]
```diff
@@ -30,6 +30,48 @@ class purchase_line_invoice(osv.osv_memory):
     _name = 'purchase.order.line_invoice'
     _description = 'Purchase Order Line Make Invoice'
 
+    def multiple_order_invoice_notes(self, orders):
```
Comment (nim-odoo) [inline addons/purchase/wizard/purchase_line_invoice.py]: I would get rid of this function and replace by `" \n".join(order.notes for order in orders)`

File: addons/purchase/wizard/purchase_line_invoice.py [hunk 2]
```diff
@@ -30,6 +30,48 @@ class purchase_line_invoice(osv.osv_memory):
...
+        notes = ""
+        for order in orders:
+            notes += "%s \n" % order.notes
+        return notes
+
+    def make_invoice_by_partner(self, cr, uid, partner, orders, lines_ids,
```
Comment (nim-odoo) [inline addons/purchase/wizard/purchase_line_invoice.py]: I think a private method is enough.

**Comment by adrienpeiffer** [comment]
Hi @nim-odoo,
Thanks for the review!

**Comment by adrienpeiffer** [comment]
Thanks @nim-odoo !

**Pull request closed without merging**